app/controllers/blogs_controller.rb

```diff
@@ -0,0 +1,60 @@
+class BlogsController < ApplicationController
+  before_action  :set_blog, only: [:show, :edit, :update, :destroy]
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.